src/modules/folder/folder.model.ts

```diff
@@ -54,11 +56,16 @@ export class FolderModel extends Model implements FolderAttributes {
   @BelongsTo(() => UserModel)
   user: UserModel;
 
+  @HasOne(() => PrivateSharingFolderModel, {
+    constraints: false,
+  })
+  privateSharingFolder: PrivateSharingFolderModel;
+
   @Column
   encryptVersion: '03-aes';
 
-  @Index
-  @Column
+  @Index('plain_name_index')
+  @Column({ field: 'plain_name' })
```

Comment:
Is `plain_name` ref strictly required? The underscored: true manages this, so I find this detail too redundant and also another thing prone to errors and required to be maintained.